spark/src/main/scala/geotrellis/spark/buffer/Implicits.scala

```diff
@@ -35,11 +36,11 @@ trait Implicits {
 
   implicit class withBufferTilesMethodsWrapper[
     K: SpatialComponent: ClassTag,
-    V <: CellGrid: Stitcher: ClassTag: (? => CropMethods[V])
```

Comment:
`TilePrototypeMethods` don't appear to be required anymore. Which is good because they would break bin api.